src/applications/bmqstoragetool/bmqstoragetool.m.cpp

```diff
@@ -57,6 +57,12 @@ parseArgs(CommandLineArguments& arguments, int argc, const char* argv[])
          "path to a .bmq_csl file",
          balcl::TypeInfo(&arguments.d_cslFile),
          balcl::OccurrenceInfo::e_OPTIONAL},
+        {"print-mode",
+         "print mode",
+         "can be one of the following: [HUMAN, JSON_PRETTY, JSON_LINE]. "
```

Comment:
Does the tool support lower-case values for this parameter? If not, it would be handy to have.